ui/src/components/DetectionsTable.tsx

```diff
@@ -111,7 +111,10 @@ export default function DetectionsTable({
         <TableBody>
           {detections
             .slice()
-            .sort((a, b) => a.id.localeCompare(b.id))
+            .sort(({ timestamp: a }, { timestamp: b }) =>
+              // Sort by timestamp, low to high
+              a == b ? 0 : a < b ? -1 : 0,
```

Comment:
```suggestion
              a === b ? 0 : a < b ? -1 : 0,
```

Reply:
But also, are these `string` or `Date`? Typescript say they should be `Date` but I don't think we ever deserialize, they look like ISO 8601 strings to me. I think we should figure out deserialization, it's safer and then you could just subtract to sort.

Don't want to bog down your PR though, can do it as a separate issue

Reply:
Actually, how about just manually doing `new Date()` for now, until we get deserialization working?

Reply:
Done!